mozetl/clientsdaily/rollup.py

```diff
@@ -114,6 +213,36 @@ def get_partition_count_for_writing(is_sampled):
     return 25
 
 
+def monthwise_main(date, input_bucket, input_prefix, output_bucket,
```

Comment:
Where is this getting called?

Reply:
It's not. I left the monthwise code in case someone wanted to use it. If y'all would rather I delete it that's fine by me.

Reply:
Do you think this would be the best way to perform backfill on large batches of `main_summary`? It could be useful when clients-daily gets ramped up from 1%.

Reply:
I think at 100% monthly almost certainly won't work due to exponential growth in the work done by groupBy(client_id), so come to think of it we should probably just delete it.